pkgs/applications/audio/vital/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, unzip
+
+, alsa-lib
+, libjack2
+, curl
+, xorg
+, libGL
+, freetype
+}:
+
+let
+  pkg_url = version: "https://builds.vital.audio/VitalAudio/vital/${builtins.replaceStrings ["."] ["_"] version}/VitalInstaller.zip";
```

Comment:
There's no reason to have this here, please remove it and use it directly in `fetchurl`.